.github/workflows/openapi-generated-types.yml

```
@@ -45,6 +48,18 @@ jobs:
           PYTHONPATH: .
         run: uv run python scripts/generate_openapi_types.py
 
+      - name: Commit regenerated artifacts for this pull request
+        if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository && github.actor != 'github-actions[bot]'
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)